src/Components/Users/Users.module.css

```diff
@@ -15,29 +16,28 @@
   background-color: #922e25b2;
 }
 
-
 .blockBtn {
-  color: #121010;
-  background: rgb(255, 255, 255);
-  cursor: pointer;
   font-size: 16px;
   text-align: center;
   border: none;
   border-radius: 15px;
+  background-color: #ef3f51;
```

Comment:
This is good , but we can make it better by using the hsl color format and reducing the lightness as below:
![hex colors to hsl](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/804bf6ba-af46-459a-93be-168c81c0cfa8)